pkgs/applications/networking/mailreaders/evolution/evolution-ews/default.nix

```diff
@@ -32,6 +35,18 @@ stdenv.mkDerivation rec {
     pkg-config
   ];
 
+  patches = [
```

Comment:
We prefer having `patches` just after `src`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.